modules/ivc_champva/app/services/ivc_champva/attachments.rb

```diff
@@ -12,13 +12,11 @@ def handle_attachments(file_path)
 
       if attachments.count.positive?
         attachments.each_with_index do |attachment, index|
-          attachment_extension = File.extname(attachment)
-
           new_file_name =
             if attachment.include?('_additional_')
-              "#{uuid}_#{File.basename(attachment, '.*')}#{index + 1}#{attachment_extension}"
+              "#{uuid}_#{File.basename(attachment, '.*')}#{index + 1}.pdf"
             else
-              "#{uuid}_#{form_id}-tmp#{index + 1}#{attachment_extension}"
+              "#{uuid}_#{form_id}-tmp#{index + 1}.pdf"
             end
```

Comment:
For detail, when an attachment is of type PDF, the ConvertToPDF service just returns the `tmp/58345983453` when its anything else it gets hardcoded to something like `tmp/142342343242.pdf`. Also, we convert everything to a PDF so we don't need a dynamic file extension feature here. Add back somewhat original logic.